### DISCUSSION TITLE
Project Part 3 , "Running the code" clarification

### DISCUSSION THREAD

**Discussion by bethccore** [IBAHCM/RPiR#558] [Q&A for project] [answered]
Hi Richard @richardreeve ,

I was just wanting to clarify, in "Running the code" for part 3 of the project, it states: "Run the two new functions again to make sure there are no errors". 

Does this mean to run the new Simpson index and species richness functions and compare the results to the answers from the Project 2 functions? And if so, do we do this for all 9 populations? 

Or does it just mean to run the Shannon entropy and Berger-Parker functions and check it works smoothly?

Also, do we include all of our 'Running the code' checks in the demo? 

Thank you in advance for your help!

**Comment by richardreeve** [comment, marked as answer]
Hi @bethccore - previously the Shannon entropy and Berger-Parker functions would very likely have generated errors, but now they should not, so you should run them to make sure they don't error. If they do, then you won't be able to create a demo. The running the code section is just to check it's working, but what I want in the demo is just what I talk about in the demo section.

**Reply by bethccore** [reply]
Ok, thank you!
